docs/05-technical-reference/00-custom-resources/ac-01-application.md

```diff
@@ -36,34 +36,77 @@ spec:
+<!-- The table below was generated automatically -->
+<!-- Some special tags (html comments) are at the end of lines due to markdown requirements. -->
+<!-- The content between "TABLE-START" and "TABLE-END" will be replaced -->
+<!-- TABLE-START -->
+<!-- Application v1alpha1 applicationconnector.kyma-project.io -->
+| Parameter         | Description                                   |
+| ---------------------------------------- | ---------|
+| **spec.accessLabel** |  |
+| **spec.compassMetadata** |  |
+| **spec.compassMetadata.applicationId** |  |
+| **spec.compassMetadata.authentication** |  |
+| **spec.compassMetadata.authentication.clientIds** |  |
+| **spec.description** | Describes the connected Application. |
+| **spec.displayName** |  |
+| **spec.encodeUrl** | Allows for URL encoding. If set to `false`, your URL segments stay intact. |
+| **spec.group** |  |
+| **spec.labels** | Defines the labels of the Application. |
+| **spec.longDescription** |  |
+| **spec.providerDisplayName** |  |
+| **spec.services** | Contains all services that the Application provides. |
+| **spec.services.authCreateParameterSchema** | New fields used by V2 version |
+| **spec.services.description** |  |
+| **spec.services.displayName** |  |
+| **spec.services.entries** | Contains the information about the APIs and events that the service offered by the Application provides. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md), you have to provide it yourself.
+ |
+| **spec.services.entries.accessLabel** | Specifies the label used in Istio rules in Application Connector. This field is required for the API entry type.
+ |
+| **spec.services.entries.apiType** |  |
+| **spec.services.entries.centralGatewayUrl** | Specifies the URL of Application Gateway. Internal address resolvable only within the cluster. This field is required for the API entry type. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md), you have to provide it yourself.
+ |
+| **spec.services.entries.credentials** |  |
+| **spec.services.entries.credentials.authenticationUrl** |  |
+| **spec.services.entries.credentials.csrfInfo** |  |
+| **spec.services.entries.credentials.csrfInfo.tokenEndpointURL** |  |
+| **spec.services.entries.credentials.secretName** |  |
+| **spec.services.entries.credentials.type** |  |
+| **spec.services.entries.gatewayUrl** |  |
+| **spec.services.entries.id** |  |
+| **spec.services.entries.name** | New fields used by V2 version |
```

Comment:
`New fields used by V2 version` conveys zero information. `Entry.name` allows to support API Bundles functionality provided by Compass. APIBundle is a set of APIs that share the same credentials. The API Bundle have both name and identifier that are used by the Compass Runtime Agent to set `spec.services.name` field. Each API in the bundle has name that is used by the Compass Runtime Agent to set  `spec.services.entries.name`.